README.md

```diff
@@ -1,29 +1,32 @@
 # mondolib
-Python library for mondo QC
-
-This has intentionally bespoke and mondo-unique checks
+Python library for Mondo.
 
+## Prerequisites
```

Comment:
I saw that OAK was required, so I made a 'prereqs' subsection, and I added the bit about inputs under here as well.